tools/cldr-apps/js/src/esm/cldrForum.js

```diff
@@ -954,15 +961,19 @@ function makeOneReplyButton(post, postType, label) {
     "addPostButton btn btn-default btn-sm"
   );
   cldrDom.listenFor(replyButton, "click", function (e) {
-    openPostOrReply({
-      /*
-       * Don't specify locale/xpath/subject/value/open for reply. Instead they will be set to
-       * match the original post in the thread.
-       */
-      replyTo: post.id,
-      replyData: post,
-      postType: postType,
-    });
+    if (postType === "Agree") {
+      cldrVoteAgree.vote(post.locale, post.xpath, post.value);
```

Comment:
calling cldrVoteAgree.vote here is the essential change in this file

Reply:
That makes it look easy !